web/src/test/java/org/openmrs/web/filter/update/UpdateFilterModelTest.java

```diff
@@ -9,49 +9,50 @@
  */
 package org.openmrs.web.filter.update;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.openmrs.liquibase.LiquibaseProvider;
-import org.openmrs.util.DatabaseUpdater;
-import org.openmrs.util.DatabaseUpdater.OpenMRSChangeSet;
-import org.openmrs.util.DatabaseUpdaterLiquibaseProvider;
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.core.classloader.annotations.PowerMockIgnore;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
-
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.collection.IsEmptyCollection.empty;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.mockito.Matchers.any;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.when;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.commons.lang3.reflect.FieldUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.openmrs.liquibase.ChangeLogDetective;
+import org.openmrs.liquibase.ChangeLogVersionFinder;
+import org.openmrs.liquibase.LiquibaseProvider;
+import org.openmrs.test.jupiter.BaseContextSensitiveTest;
+import org.openmrs.util.DatabaseUpdater;
+import org.openmrs.util.DatabaseUpdater.OpenMRSChangeSet;
+import org.openmrs.util.DatabaseUpdaterLiquibaseProvider;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.reflect.Whitebox;
+
 /**
  * Tests {@link UpdateFilterModel}.
  */
-@RunWith(PowerMockRunner.class)
-@PrepareForTest(DatabaseUpdater.class)
-@PowerMockIgnore({ "com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*", "javax.management.*", "org.w3c.dom.*" })
-public class UpdateFilterModelTest {
+public class UpdateFilterModelTest  extends BaseContextSensitiveTest {
```

Comment:
i had some error with the database connection that is why i extends BaseContextSensitiveTest